sms_sponsorship/webapp/src/App.js

```diff
@@ -99,81 +95,80 @@ class Main extends React.Component {
                 <div>
                     {topAppBar}
                     <div>
-                        <Message text={t("error_noRequestID")}/>
+                        <Error appContext={this} text="Error : no SMS request ID. Please send an other SMS."/>
```

Comment:
Text should be placed in the translations and use here the translate function. See how it's done elsewhere. This text is already existing in the translations so you just need to lookup its id in the file i18n.js and fetch it: `t("error_noRequestID")`